### PR TITLE
add link to minecraft from wikipedia

### PR DESCRIPTION
**Pull request by fz0718**
Please review this, @casey-collab

### CODE REVIEW

File: english/wikipedia/wiki.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ You are so impressed with Wikipedia, you decide that you might even like to
 [donate](http://wikimediafoundation.org/wiki/Support_Wikipedia/en) some money to help keep Wikipedia running.
 
 Until you get bored and go explore [here](https://www.urbandictionary.com) instead.
+
+Then after you explore the website, you learn a few neat tricks. 
+
+[Play minecraft](english/you-are-in-minecraft/minecraft.md)
```
Comment (casey-collab) [inline english/wikipedia/wiki.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by fz0718** [comment]
Can you look at it again, @casey-collab ?

**Comment by casey-collab** [comment]
Looks great! :checkered_flag:

**Pull request merged**